Gruntfile.js

```diff
@@ -34,154 +34,135 @@ module.exports = function (grunt) {
                 sourceDir: 'templates/twig-source',
                 sourceFiles: '<%= config.templates.sourceDir %>/**/*.html',
                 generatedDir: 'templates/generated',
-                generatedFiles: '<%= config.templates.generatedDir %>/**/*.html'
+                generatedFiles: '<%= config.templates.generatedDir %>/**/*.html',
             },
 
             lintFiles: {
                 node: ['Gruntfile.js'],
                 es: ['<%= config.srcScriptsDir %>'],
-                tests: ['<%= config.testsDir %>']
-            }
-        }
+                tests: ['<%= config.testsDir %>'],
+            },
+        },
 
     });
 
     grunt.config.merge({
         watch: {
             es: {
                 files: [
-                    '<%= config.srcScriptsDir %>/**/*.es.js',
-                    '<%= config.compiledScriptsDir %>/**/templates.js'
+                    '<%= config.srcScriptsDir %>/**/*.js',
+                    '<%= config.compiledScriptsDir %>/**/templates.js',
                 ],
                 tasks: [
                     'compilejs',
-                    'uglify'
-                ]
+                    'uglify',
+                ],
             },
             swig: {
                 files: [
-                    '<%= config.templates.sourceFiles %>'
+                    '<%= config.templates.sourceFiles %>',
                 ],
                 tasks: [
-                    'orderedSwig'
-                ]
+                    'orderedSwig',
+                ],
             },
             ngtemplates: {
                 files: ['<%= config.templates.generatedFiles %>'],
-                tasks: ['ngtemplates']
-            }
-        },
-        browserify: {
-            all: {
-                files: {
-                    '<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'
-                }
-            }
+                tasks: ['ngtemplates'],
+            },
         },
+        browserify: {all: {files: {'<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'} } },
```

Comment:
Oh, so is it enforcing you _must_ put this on oneline? That you're not allowed to indent it all? That's a shame. I thought the rule would allow us to put on oneline if we wanted, but wouldn't enforce it. I think it's more important to be able to
```js
{
    a: {
        b: {
            c: null
        }
    }
}
```
than be allowed to
```js
{a: {b: {c: null}}}
```